packages/connect/src/__mocks__/client.js

```diff
@@ -5,8 +5,8 @@
 import { JSDOM } from 'jsdom';
 const dom = new JSDOM();
 
-global.document = dom.window.document;
-global.window = dom.window;
+window.document = dom.window.document;
+window.window = dom.window;
```

Comment:
This looks a little bit strange. Shouldn't this be `window = dom.window;`?
This is not a blocker (if this will create an  issue tests will show it)

Reply:
It's not just strange, it's wrong! I changed it hastily to fix linting errors, the linter is now configured for browser environment everywhere instead of a mixture of node and browser.  This is correct since jest is configured with the default jsdom (browser) env.  Interestingly I did run all the tests of connect after I made this change and everything passed.  I suspect that this is because the jest jsdom env automatically configures a window and document for you, so maybe this bit of the mock isn't required at all - I will experiment and see.

Reply:
I say that this is strange cause I run the tests and they actually passed (not lint) - thats what surprised me a bit

Reply:
>  Shouldn't this be window = dom.window;

No window is the global object and replacing the global object is a really bad practice.  It might even screw with jest since jest is also modifying the global object - I'll have a play with various options and see what works.

Reply:
> It's not just strange, it's wrong! I changed it hastily to fix linting errors, the linter is now configured for browser environment everywhere instead of a mixture of node and browser. This is correct since jest is configured with the default jsdom (browser) env. Interestingly I did run all the tests of connect after I made this change and everything passed. I suspect that this is because the jest jsdom env automatically configures a window and document for you, so maybe this bit of the mock isn't required at all - I will experiment and see.

My suspicion was right, this JSDOM initialisation isn't needed as the jest jsdom environment already does this for us. Fixed by removing it.